data-to-sqe/qwb-to-sqe/Common/SshDatabase.cs

```diff
@@ -8,11 +8,9 @@ namespace qwb_to_sqe.Common
 {
     public class SshDatabase
     {
-        private SshClient _sshClient;
-        public MySqlConnection connection { get; private set; }
+        private readonly IConfiguration _configuration;
         private ForwardedPortLocal _port;
-        private IConfiguration _configuration;
-
+        private SshClient _sshClient;
```

Comment:
This may mean opening a connection takes longer. I think we should look at the entire "connecting to database" part of the backend - it gives you a lot of issues in tests, which shouldn't really be the case. Also, is there connection pooling?

Reply:
I will write an issue on Redmine so we take some time to investigate this.

Reply:
Ah, I didn't look closely enough.  Ingo is using this to connect SQE and QWB for periodic synchronization, so there the cost of setting up the connection is minimal in comparison to the need for a secure connection.  I think this would only run once a day or something like that, I will check with him.

By the way, I don't know why but the main database issue I had was with the naming of a cloned edition directly after creating it.  In the last PR I integrated the renaming directly in the cloning code instead of the original two-step process (clone the edition, then in another transaction rename it), no more deadlock failures.  I would like to know more about the whole connection system.  Our repos subclass a DbConnectionBase that provides its own OpenConnection(), I see other people also using DI for a transient IDbConnection.  It is a complete mystery to me how any of this performs thread pooling, and nobody seems to be discussing it. After looking at it closely I would think in our setup that each of our repos, which use transient DI, would create a new connection to the database for each individual request. I set up a little log to check and this appears to be the case.  That seems really wasteful, shouldn't we have a long running connection that pulls cursors from a pool?